apps/game/server/contacts/contacts.controller.ts

```diff
@@ -1,7 +1,29 @@
-import { Contact, ContactDeleteDTO, ContactEvents, PreDBContact } from '@typings/contact';
+import { config } from '../config';
+import { Contact, ContactDeleteDTO, ContactEvents, PreDBContact, ContactPay } from '@typings/contact';
+import PlayerService from '../players/player.service';
 import ContactService from './contacts.service';
 import { contactsLogger } from './contacts.utils';
 import { onNetPromise } from '../lib/PromiseNetEvents/onNetPromise';
+const exps = global.exports;
+
+onNetPromise<ContactPay, Contact>(ContactEvents.PAY_CONTACT, (reqObj, resp) => {
+  PlayerService.getIdentifierByPhoneNumber(reqObj.data.number, true).then((response) => {
```

Comment:
This should be behind an optional config

Reply:
again this already is from the client side